<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Codys-Mac.local" ignored="0" name="Command line test 895e792c-8e49-4138-bcc2-802a15ed3287" tests="2" failures="2" timestamp="28 Mar 2018 00:41:07 GMT" time="754.469" errors="0">
  <testcase name="prankcaller" time="298.975" classname="test.tests.launchApp.Tests">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at test.tests.launchApp.Tests.prankcaller(Tests.java:232)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- prankcaller -->
  <testcase name="launchApp" time="299.001" classname="test.tests.launchApp.Tests">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at test.tests.launchApp.Tests.launchApp(Tests.java:429)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- launchApp -->
</testsuite> <!-- Command line test 895e792c-8e49-4138-bcc2-802a15ed3287 -->
